lib/fob.ex

```diff
@@ -170,6 +180,63 @@ defmodule Fob do
     )
   end
 
+  # this function is used for comparing the ID page-break, which is a break
+  # that describes the values on the primary key of the table
+  # this assumes that primary key values must not be nil
+  defp apply_basic_comparison(page_break, comparison_strictness)
```

Comment:
the difference between all of these clauses is the comparator, one of `>`, `<`, `>=`, or `<=` from [`Ecto.Query.API`](https://hexdocs.pm/ecto/Ecto.Query.API.html#content)

unfortunately there isn't a good way I've seen to quote those in